style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+}
+#header {
+  width: 100%;
+  margin: 10px 10px 10px 10px;
+  background: lightgreen;
```

Comment:
Please for the property Background-color should be complete, if you only want the background color bacause this avoids some errors on your page.